proto/osmosis/protorev/v1beta1/params.proto

```diff
@@ -2,11 +2,17 @@ syntax = "proto3";
 package osmosis.protorev.v1beta1;
 
 import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
 
 option go_package = "github.com/osmosis-labs/osmosis/v15/x/protorev/types";
 
 // Params defines the parameters for the module.
 message Params {
-  // Boolean whether the module is going to be enabled
-  bool enabled = 1;
+  // Boolean whether the protorev module is enabled.
+  bool enabled = 1 [ (gogoproto.moretags) = "yaml:\"enabled\"" ];
+  // The admin account (settings manager) of the protorev module.
+  string admin = 2 [
+    (gogoproto.moretags) = "yaml:\"admin\"",
+    (cosmos_proto.scalar) = "cosmos.AddressString"
+  ];
```

Comment:
Moved the `admin` into params so that we can support governance proposals through both the `x/protorev` proposal handlers and the gov params change handler.